app/process/handler/crypto-transfer/crypto-transfer_test.go

```diff
@@ -0,0 +1,192 @@
+	topicSubmissionMessageBytes := []byte{0x12, 0xe3, 0x1, 0xa, 0x1e, 0x30, 0x2e, 0x30, 0x2e, 0x39, 0x39, 0x36, 0x36, 0x31, 0x2d, 0x31, 0x36, 0x31, 0x33, 0x33, 0x38, 0x39, 0x31, 0x32, 0x31, 0x2d, 0x31, 0x33, 0x35, 0x36, 0x33, 0x31, 0x34, 0x36, 0x31, 0x12, 0x2a, 0x30, 0x78, 0x37, 0x63, 0x46, 0x61, 0x65, 0x32, 0x64, 0x65, 0x46, 0x31, 0x35, 0x64, 0x46, 0x38, 0x36, 0x43, 0x66, 0x64, 0x41, 0x39, 0x66, 0x32, 0x64, 0x32, 0x35, 0x41, 0x33, 0x36, 0x31, 0x66, 0x31, 0x31, 0x32, 0x33, 0x46, 0x34, 0x32, 0x65, 0x44, 0x44, 0x18, 0x90, 0x4e, 0x22, 0xd, 0x31, 0x31, 0x32, 0x36, 0x32, 0x32, 0x31, 0x32, 0x33, 0x37, 0x32, 0x31, 0x31, 0x2a, 0x82, 0x1, 0x35, 0x37, 0x62, 0x61, 0x30, 0x33, 0x65, 0x63, 0x35, 0x39, 0x36, 0x39, 0x30, 0x38, 0x61, 0x66, 0x66, 0x61, 0x64, 0x65, 0x62, 0x32, 0x32, 0x62, 0x32, 0x61, 0x34, 0x37, 0x31, 0x64, 0x66, 0x62, 0x61, 0x66, 0x61, 0x34, 0x35, 0x61, 0x62, 0x39, 0x66, 0x62, 0x63, 0x31, 0x66, 0x62, 0x35, 0x66, 0x37, 0x31, 0x61, 0x31, 0x34, 0x38, 0x35, 0x65, 0x61, 0x65, 0x61, 0x63, 0x33, 0x32, 0x39, 0x35, 0x30, 0x38, 0x63, 0x31, 0x33, 0x66, 0x64, 0x64, 0x35, 0x38, 0x61, 0x62, 0x34, 0x32, 0x64, 0x64, 0x34, 0x61, 0x38, 0x37, 0x64, 0x63, 0x66, 0x37, 0x63, 0x65, 0x39, 0x39, 0x38, 0x33, 0x33, 0x35, 0x64, 0x32, 0x36, 0x39, 0x36, 0x36, 0x38, 0x35, 0x66, 0x37, 0x34, 0x61, 0x32, 0x38, 0x62, 0x31, 0x34, 0x33, 0x33, 0x31, 0x64, 0x30, 0x35, 0x32, 0x35, 0x36, 0x31, 0x62, 0x64, 0x36, 0x36, 0x37, 0x31, 0x62}
+
+	return ctm, topicID, accID, cryptoTransferPayload, topicSubmissionMessageBytes
+}
+
+func Test_Handle_Not_Initial_Transaction(t *testing.T) {
+	ctm, topicID, accID, cryptoTransferPayload, topicSubmissionMessageBytes := GetTestData()
+	ctHandler, transactionRepo, hederaNodeClient, hederaMirrorClient := InitializeHandler()
+
+	proto.Unmarshal(cryptoTransferPayload, &ctm)
+
+	expectedTransaction := hedera.TransactionID{
+		AccountID:  accID,
+		ValidStart: time.Time{},
+	}
+
+	tx := &transaction.Transaction{
+		Model:          gorm.Model{},
+		TransactionId:  ctm.TransactionId,
+		EthAddress:     ctm.EthAddress,
+		Amount:         ctm.Amount,
+		Fee:            ctm.Fee,
+		Signature:      signature,
+		SubmissionTxId: submissionTxID,
+		Status:         txRepo.StatusCompleted,
+	}
+
+	txs := txn.HederaTransactions{
+		Transactions: []txn.HederaTransaction{},
+	}
+
+	transactionRepo.On("GetByTransactionId", ctm.TransactionId).Return(tx, nil)
+	transactionRepo.On("UpdateStatusSignatureSubmitted", ctm.TransactionId, submissionTxID, signature).Return(nil)
+	hederaNodeClient.On("SubmitTopicConsensusMessage", topicID, topicSubmissionMessageBytes).Return(&expectedTransaction, nil)
+	hederaMirrorClient.On("GetAccountTransaction", submissionTxID).Return(&txs, nil)
+
+	ctHandler.Handle(cryptoTransferPayload)
+
+	transactionRepo.AssertNotCalled(t, "UpdateStatusSignatureSubmitted", ctm.TransactionId, submissionTxID, signature)
```

Comment:
I think that it would be beneficial to assert that:
- `cth.hederaNodeClient.SubmitTopicConsensusMessage` is not called as-well
- `GetAccountTransaction` is not called as-well
This would verify that `handleTopicSubmission` and `checkForTransactionCompletion` were called